### PR TITLE
Update continental extension cookbook

### PR DESCRIPTION
**Pull request by naliboff**
A few changes to the continental extension cookbook to fix the error in the radiogenic heating term (switched to compositional heating model) and a few additional minor parameter file adjustments.

### CODE REVIEW

**Comment by gassmoeller** [comment]
Seems reasonable. Interesting that the results change quite a bit although the value should be the same. Do you have an explanation for that?

**Comment by naliboff** [comment]
@gassmoeller -Are you looking at the differences between the high resolution images (strain rate at 10 Myr snapshot)? The shear band patterns and system overall starts evolving quite quickly once extension gets going. With the slightly higher geotherm, the system is simply at a different "state" and due to the non-linearity things change quickly. Honestly, I'm surprised it did not change more. Running the same setup with different codes (similar methods) but nearly identical initial conditions sometimes gives noticeably different results. I guess "the results change quite a bit" is somewhat up to interpretation. Working on trying to quantifying these types changes at the moment :)

**Comment by gassmoeller** [comment]
I guess my question was: Why are we at a slightly higher geotherm, if the radioactive heating before and now are the same (1.e5e-6 W/m^3)?
Is there a noticeable drop in heating over 10 Myr due to radioactive decay in the old version? Or is 1.5e-6 in the old version the *initial* value that is reduced by the assumed decay over 4.5 billion years?

File: doc/manual/manual.tex [hunk 1]
```diff
@@ -7945,17 +7945,17 @@ \subsubsection{Continental extension}
 \begin{figure}
 \centering
 \includegraphics[width=\textwidth]{cookbooks/continental_extension/continental_extension_cookbook_strainrate_5e6yr.png}
-\caption{\it Strain rate ($s^{-1}$) after 5e6 years of extension. The black line marks the 500 $^{o}C$ isotherm.} 
+\caption{\it Strain rate ($s^{-1}$) after 5e6 years of extension. The black line marks the 550 $^{o}C$ isotherm.}
```
Comment (bangerth) [inline doc/manual/manual.tex]: While you're there, can you change this to `$550^\circ$C`? That creates the proper "degree" symbol. Same for the other figures below.

**Comment by naliboff** [comment]
Ah, sorry for the confusion. The radioactive heating before was not correct as the the initial concentration should have been 1e6 instead of 1

File: doc/manual/manual.tex (hunk 1 above)
Comment (naliboff) [inline doc/manual/manual.tex]: :+1:

**Comment by gassmoeller** [comment]
Now it makes sense :smile:. Ready to merge when Wolfgang's comment is addressed.

**Pull request merged**